### ISSUE TITLE
[Example Request]: Polygon

### ISSUE THREAD

**Issue by mortmoe** [tldraw/tldraw#4919]
### What's the example?

How would you go about to create a polygon, or irregular shape. I can create the shape with the line-tool and get it to "snap" close, but the opening is still there, and it's still just a line :) I'd like to be able to fill it (and maybe do some calculations, ie, calculate area and circumference) :) Any ideas?

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by roychowd** [comment]
Hey did you figure this out? I'm also trying to work on a polygon tool

**Comment by mortmoe** [comment]
Nope. Haven't had time to look any more into it either. Was hoping someone had an example. I'm not currently using this component but considering it as I will need to make some kind of drawing input for my users soon. Beeing able to draw polygons and polylines would be a requirement though, freehand would just not be any good for my case. I guess my polygons and polylines should also print the angles so might have to create something from scratch 🤣

**Comment by roychowd** [comment]
lol, guess you and I are in the same boat. I'll let you know if i made any progress on this

**Comment by mortmoe** [comment]
Sounds good, I'll do the same. Depends on my employers ever changing priority list tbh. 🤣

**Comment by marpstar** [comment]
@mortmoe @roychowd I'm working on something with similar requirements. Either of you able to share what you've got? Happy to collaborate on a solution.
